pkg/server/store/pkg/schema.go

```diff
@@ -128,3 +143,40 @@ func updateInstallScript(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	_, _ = w.Write([]byte("{}"))
 }
+
+func cancelHandler(w http.ResponseWriter, r *http.Request) {
+	apiContext := types.GetAPIContext(r.Context())
+	defer r.Body.Close()
```

Comment:
Do we need to close request body here?

Reply:
According to https://pkg.go.dev/net/http#Request, I think it doesn't need be closed.

Reply:
Pushed a new commit to remove all the unnecessary `request.Body.close` operation.

Reply:
Missing remove `request.Body.Close` here.

Reply:
Updated